core/arch/asm_defines.asm

```diff
@@ -196,11 +198,15 @@ ASSUME fs:_DATA @N@\
 #  define PUSH_SEH(reg) push reg @N@ .pushreg reg
 /* Push a volatile register or an immed in prolog: */
 #  define PUSH_NONCALLEE_SEH(reg) push reg @N@ .allocstack 8
+# define ADD_STACK_ALIGNMENT sub REG_XSP, FRAME_ALIGMMENT - ARG_SZ @N@ .allocstack 16
+# define RESTORE_STACK_ALIGNMENT add REG_XSP, FRAME_ALIGMMENT - ARG_SZ @N@ .allocstack 16
```

Comment:
The epilogue has no directives, just a strict order (add rsp, then pops, then ret).

Reply:
Removed .allocstack.